### PR TITLE
standardising version above & below

### PR DESCRIPTION
**Pull request by petersgiles**
@SimonCropp is this what you wanted?

### CODE REVIEW

File: servicecontrol/creating-config-file.md [hunk 1]
```diff
@@ -134,9 +134,9 @@ Default: `350`.
 
 #### ServiceControl/MaxBodySizeToStore
 
-Up until Version 1.6 ServiceControl only stores bodies of audit messages that are smaller than 100Kb.
+This setting exists in Version 1.6 and above. It allows the upper limit on body size to be configured. 
 
-Version 1.6 introduced this setting which allows the upper limit on body size to be configured. 
+In Version 1.5.3 and below ServiceControl only stores bodies of audit messages that are smaller than 100Kb.
```
Comment (SimonCropp) [inline servicecontrol/creating-config-file.md]: `1.5.x` in case we release a `1.5.4`

File: servicecontrol/creating-config-file.md [hunk 2]
```diff
@@ -115,9 +115,9 @@ Type: int
 
 Default: `720` (30 days). 
 
-Prior to Version 1.8.3 the valid range for this setting was `24` (1 day) through to `1440` (60 days)
+In Versions 1.8.2 and below the valid range for this setting was `24` (1 day) through to `1440` (60 days)
 
-From Version 1.8.3 the upper limit has been removed to allow for longer retention.  This was done to allow customers with low volumes of messages to retain them longer.  Setting this value too high can cause the embeddeded RavenDB to become large and unresponsive when indexing.  See [Capacity and Planning](capacity-and-planning.md)
+In Versions 1.8.3 and above the upper limit has been removed to allow for longer retention.  This was done to allow customers with low volumes of messages to retain them longer.  Setting this value too high can cause the embeddeded RavenDB to become large and unresponsive when indexing.  See [Capacity and Planning](capacity-and-planning.md)
```
Comment (SimonCropp) [inline servicecontrol/creating-config-file.md]: `1.8.x`

File: servicecontrol/creating-config-file.md (hunk 1 above)
Comment (petersgiles) [inline servicecontrol/creating-config-file.md]: do we have a convention where '.x' means the latest?

Comment (SimonCropp) [inline servicecontrol/creating-config-file.md]: yes

File: servicecontrol/creating-config-file.md (hunk 2 above)
Comment (petersgiles) [inline servicecontrol/creating-config-file.md]: how does the '.x' convention work here when we are in the middle of a range?

Comment (SimonCropp) [inline servicecontrol/creating-config-file.md]: actually u r correct. ignore that comment

**Pull request merged**